### PR TITLE
fix: build fixes for non-linux platforms

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

/area libscap
/area libsinsp
/area tests

**Does this PR require a change in the driver versions?**

**What this PR does / why we need it**:

TODO; nothing to see here, just testing things :) 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: userspace/chisel/chisel.h [hunk 1]
```diff
@@ -18,9 +18,10 @@ limitations under the License.
 #pragma once
 
 #include <sinsp.h>
-#include "filterchecks.h"
```
Comment (FedeDP) [inline userspace/chisel/chisel.h:21]: Do not include filterchecks here, forward declare the required `sinsp_filter_check` instead.

**Review by gnosek — APPROVED** [review]

File: cmake/modules/libsinsp.cmake [hunk 2]
```diff
@@ -32,7 +32,7 @@ if(NOT MINIMAL_BUILD)
 	include(cares)
 endif()
 
-set(LIBSINSP_INCLUDE_DIRS ${LIBSINSP_DIR}/userspace/libsinsp ${LIBSINSP_DIR}/common ${LIBSCAP_INCLUDE_DIRS} ${DRIVER_CONFIG_DIR})
+set(LIBSINSP_INCLUDE_DIRS ${LIBSINSP_DIR}/userspace/libsinsp ${LIBSINSP_DIR}/userspace/common ${LIBSCAP_INCLUDE_DIRS} ${DRIVER_CONFIG_DIR})
```
Comment (Andreagit97) [inline cmake/modules/libsinsp.cmake:35]: so before this change the include dir `${LIBSINSP_DIR}/common` was completely useless, ok

**Pull request merged**